test/e2e_federation/replicaset.go

```diff
@@ -170,7 +170,7 @@ var _ = framework.KubeDescribe("Federated ReplicaSet [Feature:Federation]", func
 			})
 
 			// test for rebalancing
-			It("should create replicasets and rebalance them", func() {
+			Pit("should create replicasets and rebalance them", func() {
```

Comment:
Ginkgo doc says `PIt` with a capital I - https://onsi.github.io/ginkgo/#the-spec-runner